### PR TITLE
Range fixes

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
This fixes https://github.com/quarkusio/quarkus/issues/9990

File: servlet/src/test/java/io/undertow/servlet/test/defaultservlet/DefaultServletCachedResourceTestCase.java [hunk 1]
```diff
@@ -0,0 +1,302 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
```
Comment (cescoffier) [inline servlet/src/test/java/io/undertow/servlet/test/defaultservlet/DefaultServletCachedResourceTestCase.java:3]: 2014... Someone really needs to update his IDE ;-)

Comment (stuartwdouglas) [reply servlet/src/test/java/io/undertow/servlet/test/defaultservlet/DefaultServletCachedResourceTestCase.java:3]: It was a copy/paste

**Comment by cescoffier** [comment]
Waiting for CI.

**Pull request merged**